tern/utils/cache.py

```diff
@@ -27,6 +27,11 @@
 def load():
     '''Load the cache'''
     global cache
+
+    # small guard to stop ENOENT if the cache is not missing
```

Comment:
:D Not everyone knows what ENOENT means (now I'm curious on what your traceback looks like and if you're using any tools to view it).

Reply:
Ah! sorry I'll rephrase and make it clearer :+1: